app/controllers/projects_controller.rb

```diff
@@ -48,7 +48,7 @@ def update
 
     @project.save
     flash[:notice] = I18n.t("project.update.success")
-    redirect_to project_path(@project)
+    redirect_to project_information_path(@project)
```

Comment:
How come we are redirecting to the project info path now?

Reply:
The logic is "nothing I can edit is displayed on the task list, it's all on the project information page".

It's very tenuous though. Mostly it meant I could leave tests as they were (since the relevant info is now on project information but not in the summary), and we can pass it over to research for "how do people expect editing to behave".